app/assets/stylesheets/blacklight/_pagination.scss

```diff
@@ -14,3 +14,20 @@
     flex-wrap: wrap;
   }
 }
+
+
+.link-pagination {
+  .disabled a {
+    pointer-events: none !important;
+    cursor: default;
+    color:gray;
+  }
+
+  .active span {
+    color: black;
```

Comment:
can we set `--bs-pagination-active-color` instead?  Is there a reason we need to customize this here and not in the host app?  The host app may make different choices for which colors are most appropriate.